### PR TITLE
Roadmap in mermaid form

### PR DESCRIPTION
**Pull request by richelbilderbeek**
Dear subset of the Game Design team,

An easy PR, mostly to follow our team's rules and communicate the new road map.

Here I extend the road map a bit using mermaid (see [https://mermaid-js.github.io/mermaid/#/](https://mermaid-js.github.io/mermaid/#/)). You can see how it looks at [https://github.com/tresinformal/game/blob/richel/doc/roadmap.md#roadmap](https://github.com/tresinformal/game/blob/richel/doc/roadmap.md#roadmap).

This road map is not perfect, nor intends to be. It is just a step forward. Instead of asking me to modify it to suit the Game Design team, I suggest to accept it and modify it instead.

### CODE REVIEW

File: doc/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Roadmap
+
+```mermaid
+graph TD;
+    0((Now))-->10[1. The game has a complete menu interface];
+    0-->20[2. The game keeps track of a score for each player]
+    10-->30[3. Player size]
+    0-->40[?Scoreboard]
```
Comment (TheoPannetier) [inline doc/roadmap.md:8]:
```suggestion

```

We (Oliver and I) think the scoreboard should be part of Milestone 2.

File: doc/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    20-->30
+    40-->100
+    50[?Fix or temporarily remove rockets]-->100
```
Comment (TheoPannetier) [inline doc/roadmap.md:11]:
```suggestion
    50[?. Fix or temporarily remove rockets]-->100
```

A bit OCDey, but a space may help readability :)

File: doc/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    60[?Fix or temporarily remove stun rockets]-->100
```
Comment (TheoPannetier) [inline doc/roadmap.md:12]:
```suggestion
    60[?. Fix or temporarily remove stun rockets]-->100
```

File: doc/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    70[?Fix or temporarily remove food and eating dynamics]-->100
+    30-->100((v1.0: A minimal viable game can be won/lost))
+    100-->200((?v1.x: Make game sound-complete))
```
Comment (TheoPannetier) [inline doc/roadmap.md:15]:
```suggestion
    100-->200((?. v1.x: Make game sound-complete))
```

File: doc/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    100-->300((?v1.x: Can set all options and control in the options screen))
+    100-->400((?v1.x: Add AI opponent))
+    100-->500((?v1.x: Add PvP over network))
+    100-->600((?v1.x: Make game look beautiful))
```
Comment (TheoPannetier) [inline doc/roadmap.md:19]:
```suggestion
    100-->300((?. v1.x: Can set all options and control in the options screen))
    100-->400((?. v1.x: Add AI opponent))
    100-->500((?. v1.x: Add PvP over network))
    100-->600((?. v1.x: Make game look beautiful))
```

File: doc/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+graph TD;
+    0((Now))-->10[1. The game has a complete menu interface];
+    0-->20[2. The game keeps track of a score for each player]
+    10-->30[3. Player size]
+    0-->40[?Scoreboard]
+    20-->30
+    40-->100
+    50[?Fix or temporarily remove rockets]-->100
+    60[?Fix or temporarily remove stun rockets]-->100
+    70[?Fix or temporarily remove food and eating dynamics]-->100
```
Comment (TheoPannetier) [inline doc/roadmap.md:13]:
```suggestion
    70[?. Fix or temporarily remove food and eating dynamics]-->100
```

File: doc/roadmap.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Roadmap
+
+```mermaid
+graph TD;
+    0((Now))-->10[1. The game has a complete menu interface];
+    0-->20[2. The game keeps track of a score for each player]
```
Comment (TheoPannetier) [inline doc/roadmap.md:6]:
```suggestion
    0-->20[2. The game keeps track of a score for each player using a scoreboard]
```

**Review by TheoPannetier — CHANGES_REQUESTED** [review]
We request minor changes, but this looks exciting! We had a chance to see the mermaid syntax 🧜

**Comment by richelbilderbeek** [comment]
@TheoPannetier accepted you commits :-)

**Pull request merged**